setup.py

```diff
@@ -52,7 +52,7 @@
         'PyYAML ~= 5.1',
         'requests ~= 2.22.0',
         'structlog ~= 19.1.0',
-        'tenacity ~= 5.0.4',
+        'tenacity ~= 5.0.4'
```

Comment:
Why this change? I like having the extra `,` so that you don't have to remember to add it if you add a new requirement below this one.